<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proizvodukloni</class>
 <widget class="QDialog" name="proizvodukloni">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>837</width>
    <height>535</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>837</width>
    <height>535</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Aplikacija Pekara | Uklananje Proizvoda</string>
  </property>
  <widget class="QLabel" name="NazivLabela">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Naziv:</string>
   </property>
  </widget>
  <widget class="QLabel" name="naslov">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Uklanjanje  proizvoda u magacinu:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ukloniDugme">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ukloni</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dugmeIzlazAz">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Izlaz</string>
   </property>
  </widget>
  <widget class="QComboBox" name="magacinIzbor_az">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Gram</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kilogram</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Komad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tona</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="jedMereLabela">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jedinica mere:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labela_ID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Šifra proizvoda:</string>
   </property>
  </widget>
  <widget class="QLabel" name="KolicinaLabela_az">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Količina:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nbCena">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxID">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="KolicinaLabela_az_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nabavna cena:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_naziv_az">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kolicina_az">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
